opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -37,6 +40,7 @@
 @ConditionalOnBean(Tracer.class)
 @AutoConfigureAfter(TracerAutoConfiguration.class)
 @EnableConfigurationProperties(WebTracingProperties.class)
+@ConditionalOnClass(Servlet.class)
```

Comment:
IN SB 2 it's possible to write 

`@ConditionalOnWebApplication(type = ConditionalOnWebApplication.Type.SERVLET)`